tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/SupplierSectionSteps.cs

```diff
@@ -86,16 +90,75 @@ public async Task ThenTheResponseContainsTheFollowingSupplierAddress(Table table
                 Line3 = address.Value<string>("line3"),
                 Line4 = address.Value<string>("line4"),
                 Line5 = address.Value<string>("line5"),
-                Town =  address.Value<string>("town"),
-                County =  address.Value<string>("county"),
-                Postcode =  address.Value<string>("postcode"),
-                Country =  address.Value<string>("country")
+                Town = address.Value<string>("town"),
+                County = address.Value<string>("county"),
+                Postcode = address.Value<string>("postcode"),
+                Country = address.Value<string>("country")
             };
 
             var expected = table.CreateInstance<SupplierAddressTable>();
             actual.Should().BeEquivalentTo(expected);
         }
 
+        [When(@"the user makes a request to update the supplier with order ID (.*)")]
+        public async Task WhenTheUserMakesARequestToUpdateTheSupplierWithOrderId(string orderId, Table table)
+        {
+            var supplierTable = table.CreateInstance<SupplierSectionTable>();
+            var addressTable = _context["SupplierAddress"];
+            var contactTable = _context["SupplierContact"];
+
+            var data = new
+            {
+                supplierTable.SupplierId,
+                Name = supplierTable.SupplierName,
+                Address = addressTable,
+                PrimaryContact = contactTable
+            };
+
+            await _request.PutJsonAsync(string.Format(_orderSupplierSectionUrl, orderId), data);
+        }
+
+        [When(@"the user makes a request to update the supplier with order ID (.*) with no model")]
+        public async Task WhenTheUserMakesARequestToUpdateTheSupplierWithOrderIdWithNoModel(string orderId)
+        {
+            await _request.PutJsonAsync(string.Format(_orderSupplierSectionUrl, orderId), null);
+        }
+
+        [Then(@"the supplier address for order (.*) is")]
+        public async Task ThenTheSupplierAddressForOrderIs(string orderId, Table table)
+        {
+            var address = table.CreateInstance<SupplierAddressTable>();
+
+            var addressId = (int)(await OrderEntity.FetchOrderByOrderId(_settings.ConnectionString, orderId))
+                .SupplierAddressId;
+
+            var actual = await AddressEntity.FetchAddressById(_settings.ConnectionString, addressId);
+
+            actual.Should().BeEquivalentTo(address);
+        }
+
+        [Then(@"the supplier contact for order (.*) is")]
+        public async Task ThenTheSupplierContactIdContactForOrderCIs(string orderId, Table table)

```

Comment:
I feel like that C at the end of the function name can go

Reply:
👍